x-pack/plugins/ml/server/models/job_validation/validate_model_memory_limit.ts

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import numeral from '@elastic/numeral';
+import { APICaller } from 'kibana/server';
+import { AnomalyDetectionJob } from '../../routes/schemas/anomaly_detectors_schema';
+import { validateJobObject } from './validate_job_object';
+import { calculateModelMemoryLimitProvider } from '../calculate_model_memory_limit';
+import { ALLOWED_DATA_UNITS } from '../../../common/constants/validation';
+
+// The minimum value the backend expects is 1MByte
+const MODEL_MEMORY_LIMIT_MINIMUM_BYTES = 1048576;
+
+export async function validateModelMemoryLimit(
+  callWithRequest: APICaller,
+  job: AnomalyDetectionJob,
```

Comment:
this should be a `Job`

Reply:
Changed in https://github.com/elastic/kibana/pull/60376/commits/0ec72638475bfebead195972a3034f6012011104